mitodl/open-discussions
# Import program url and dates for xPro

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #2254 

#### What's this PR do?
Adds 3 date fields to the Program model and populates these fields plus URL during xPro program import.

#### How should this be manually tested?
- Set `XPRO_CATALOG_API_URL=https://xpro-rc.odl.mit.edu/api/programs/`
- Run `course_catalog.tasks.get_xpro_data`
- Verify that the `start_date`, `end_date`, `enrollment_start`, and `url` fields are populated for the imported programs.
